xfer.py

```diff
@@ -994,11 +995,10 @@ def analyze(transfer_manifest_path: Path) -> None:
 
             local_size = (local_dir / fname).stat().st_size
             if local_size != size:
-                raise InconsistentManifest(
-                    "Local size of {} of {} does not match anticipated size {}.".format(
-                        local_size, fname, size,
-                    )
-                )
+                mismatched_filesizes[fname] = {"expected": size, "got": local_size}
+            else:
+                if fname in mismatched_filesizes:
+                    del mismatched_filesizes[fname]
```

Comment:
```suggestion
                mismatched_filesizes.pop(fname, None)
```
is a little cleaner IMO